src/application/common/logger.ts

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { inject, injectable, unmanaged } from 'inversify';
+import { LoggerWatcher } from './logger-watcher';
+
+export namespace LogLevel {
+    export const FATAL = 60;
+    export const ERROR = 50;
+    export const WARN = 40;
+    export const INFO = 30;
+    export const DEBUG = 20;
+    export const TRACE = 10;
+}
+
+export const ILoggerServer = Symbol("ILoggerServer");
```

Comment:
Would it be possible to move `ILoggerServer`,  `ILoggerClient` and `ILogLevelChangedEvent` to another file, like `logger-protocol.ts`? To me, they are implementation details of `ILogger` a client don't need to be exposed directly to them.